force-app/main/default/classes/TaskTriggerHelperTest.cls

```diff
@@ -0,0 +1,32 @@
+@isTest
+public class TaskTriggerHelperTest {
```

Comment:
`TaskTriggerTestHelper` maybe. 

this new class introduces a few things
1) API version mismatch (v56 in project vs v57 in this class)
2) while the methods cannot be called outside a test context, what about code coverage? Does your new class impact overall code coverage? (it adds new methods, but no tests)